scripts/ec2cleanup.py

```diff
@@ -129,7 +133,35 @@ def delete_unused_network_interfaces(provider_mgmt, excluded_enis, output):
         logger.exception('Exception in %r', delete_unused_network_interfaces.__name__)
 
 
-def ec2cleanup(exclude_volumes, exclude_eips, exclude_elbs, exclude_enis, output):
+def delete_stacks(provider_mgmt, excluded_stacks, stack_template, output):
+    stack_list = []
+    provider_name = provider_mgmt.kwargs['name']
+    try:
+        for stack in provider_mgmt.list_stack():
+            if (excluded_stacks and
```

Comment:
you don't need braces here

Reply:
it's for formatting purpose. conditions don't feet in 100 - each condition from new line for readability.

Reply:
ah, you're right